cime_config/config_archive.xml

```diff
@@ -30,6 +30,10 @@
     <rest_file_extension>rhist\.(INSTANT|AVERAGE|MAX|MIN)\.n(step|sec|min|hour|day|month|year)s_x\d*</rest_file_extension>
     <!-- The following matches "hi.AVGTYPE.FREQUNITS_xFREQ.TIMESTAMP.nc"-->
     <hist_file_extension>.*\.h\.(?!rhist\.).*\.nc$</hist_file_extension>
+    <rpointer>
+      <rpointer_file>rpointer.atm$NINST_STRING</rpointer_file>
+      <rpointer_content>$CASE.scream$NINST_STRING.r.$DATENAME.nc </rpointer_content>
```

Comment:
Is this telling CIME what the file content will be? Our rpointer will also contain rhist files though, no? I guess I just don't know how this var is used by cime...

Reply:
The documentation of these items is basically nonexistent. I will double check a few things.